kclvm/parser/src/lexer/mod.rs

```diff
@@ -561,10 +562,17 @@ impl<'a> Lexer<'a> {
         let start = self.pos;
 
         if self.indent_cxt.nesting > 0 {
-            self.sess.struct_span_error(
+            self.sess.struct_span_error_recovery(
                 "Unclosed nesting at the end of the file",
                 self.span(start, self.pos),
-            )
+            );
+
+            for _ in 0..self.indent_cxt.nesting {
+                buf.push(Token::new(
```

Comment:
I'm not sure it's necessary to distinguish between paren, bracket and brace here @Peefy

Reply:
Pls add more test cases about paren, bracket and brace parse.